ase/clapplugin.cc

```diff
@@ -398,10 +406,33 @@ class ClapAudioWrapper : public AudioProcessor {
     output_events_.resize (0);
   }
   void
+  update_transportinfo()
+  {
+    const auto& trans = transport();
+    const auto& tick_sig = trans.tick_sig;
```

Comment:
Please make this `auto &tick_sig`, in Anklang the type qualifiers `& * &&` are attached to variables without inbetween spaces to avoid misinterpretation if multiple variables are declared (e.g. `char *foo, *bar;`)

Reply:
Ok, I changed this.